lib/features/home/di/home_locator.dart

```diff
@@ -0,0 +1,27 @@
+import 'package:get_it/get_it.dart';
+
+import '../data/repository/home_repository_impl.dart';
+import '../data/sources/home_remote_data_source.dart';
+import '../data/sources/home_remote_data_source_impl.dart';
+import '../domain/repository/home_repository.dart';
+import '../domain/use_cases/get_home_data_use_case.dart';
+import '../presentation/cubit/home_cubit.dart';
+
+Future<void> initHomeLocator(GetIt locator) async {
+  // data sources
+  locator.registerFactory<HomeRemoteDataSource>(
```

Comment:
locator should be on on common and singleton to register all data sources and repos on time and must be lazy loaded

Reply:
Done